src/main/java/Duke.java

```diff
@@ -1,10 +1,162 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /**
+     * Number of tasks scheduled
```

Comment:
taskCounter is quite self-explanatory, the comment is probably trivial in this case.

Reply:
Okay sure, I will eliminate them.